Riot/Modules/Room/CellData/RoomBubbleCellData.m

```diff
@@ -1076,7 +1084,7 @@ - (void)loadURLPreview
 {
     // Get the last bubble component as that contains the link.
     MXKRoomBubbleComponent *lastComponent = bubbleComponents.lastObject;
-    if (!lastComponent)
+    if (!lastComponent || !lastComponent.link)
```

Comment:
Should I leave this in? I've tested it without and the change to `hasLink` does fix the issue, but wondering if leaving in an extra check for safety is a good idea.

Reply:
Yeah we can leave it.